backend/apps/volontulo/tests/views/api/offers/test_read.py

```diff
@@ -17,16 +18,7 @@ class _TestOffersReadAPIView(TestOffersCommons, APITestCase):
 
     def _test_offer_read_fields(self, offer):
         """Test read's fields of offers REST API endpoint."""
-        self.assertIsInstance(offer.pop('finished_at'), str)
-        self.assertIsInstance(offer.pop('id'), int)
-        self.assertIsInstance(offer.pop('image'), (str, type(None)))
-        self.assertIsInstance(offer.pop('location'), str)
-        self.assertIsInstance(offer.pop('organization'), str)
-        self.assertIsInstance(offer.pop('slug'), str)
-        self.assertIsInstance(offer.pop('started_at'), str)
-        self.assertIsInstance(offer.pop('title'), str)
-        self.assertIsInstance(offer.pop('url'), str)
-        self.assertEqual(len(offer), 0)
+        common.test_offer_list_fields(self, offer)
```

Comment:
I'm missing a bit of the context here, but cannot there be a test checking the response as a whole (with fields content), not just the types?